lib/js/atoms/SurveyQuestion/SurveyQuestionScale.vue

```diff
@@ -0,0 +1,240 @@
+						<survey-question-textarea
+							:id="inputId"
+							:value="elaborationValue"
+							class="surveyQuestionScale__elaborationInput"
+							:placeholder="placeholder"
+							:disabled="state === SURVEY_QUESTION_STATES.DISABLED"
+							@input="$emit('elaboration-change', $event)"
+						/>
+					</div>
+				</template>
+			</template>
+		</ds-card>
+	</div>
+</template>
+
+<style lang="scss" scoped>
+@import '../../../styles/settings/buttons';
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/media-queries';
+@import '../../../styles/settings/radiuses';
+@import '../../../styles/settings/shadows';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.surveyQuestionScale {
+	&__title {
+		@include headlineS();
+
+		display: flex;
+		// title without explanation iconButton has to be the same size as with iconButton
+		min-height: $icon-button-medium-size;
+		margin-bottom: $space-m;
+		justify-content: space-between;
+	}
+
+	&__explanation {
+		align-self: flex-start;
+	}
+
+	&__content {
+		background: $color-background-secondary;
+		border-radius: $radius-s;
+		display: flex;
+		margin-bottom: $space-xxs;
+		padding: $space-s $space-xxs;
+		overflow-x: auto;
+
+		@media #{breakpoint-s()} {
+			padding: $space-s;
+		}
+	}
+
+	&__toggle {
+		display: flex;
+		flex-grow: 1;
+		justify-content: center;
+		margin-right: $space-xxs;
+
+		&:last-child {
+			margin-right: 0;
+		}
+	}
+
+	&__toggleSeparator {
+		display: none;
+
+		@media #{breakpoint-s()} {
+			display: block;
+			flex-grow: 3;
+		}
+	}
+
+	&__separator {
+		border-bottom: none;
+		border-top: 1px solid $color-mischka-gray;
+		margin: $space-m 0 $space-s 0;
+	}
+
+	&__elaboration {
+		display: flex;
+		flex-direction: column;
+	}
+
+	&__elaborationLabel {
+		@include headlineXS();
+	}
+
+	&__elaborationInput {
+		margin: $space-xxs 0;
+	}
+}
+</style>
+
+<script lang="ts">
+import DsCard from '../Card';
+import IconButton from '../IconButton';
+import { ICON_SIZES, ICONS } from '../Icon';
+import DsButton, { BUTTON_COLORS, BUTTON_TYPES } from '../Button';
+import DsModal from '../Modal';
+import SurveyToggle, {
+	SURVEY_TOGGLE_MEANINGS,
+	SURVEY_TOGGLE_STATES,
+	SURVEY_TOGGLE_STATUSES,
+} from '../SurveyToggle';
+import { SURVEY_QUESTION_STATES } from './SurveyQuestion.consts';
+import SurveyQuestionTextarea from './SurveyQuestionTextarea.vue';
+
+export default {
+	name: 'SurveyQuestionScale',
+	components: {
+		SurveyQuestionTextarea,
+		DsCard,
+		IconButton,
+		SurveyToggle,
+		DsModal,
+		DsButton,
+	},
+	props: {
+		title: {
+			type: String,
+			required: true,
+		},
+		state: {
+			type: String,
+			default: SURVEY_QUESTION_STATES.DEFAULT,
+			validate(state) {
+				return Object.values(SURVEY_QUESTION_STATES).includes(state);
+			},
+		},
+		scaleOptions: {
+			type: Array,
+			required: true,
+			validate(scaleOptions) {
+				return scaleOptions.every((option) => typeof option === 'object');
```

Comment:
Let's enumerate all possible option params here. Related to https://github.com/bethinkpl/design-system/pull/71/files#r745576752

Reply:
[`97af32b` (#71)](https://github.com/bethinkpl/design-system/pull/71/commits/97af32b3fdc55953bdc020f1b452474f96e9f10c)